src/test/routes/logout.ts

```diff
@@ -21,11 +21,31 @@ describe('Logout receiver', () => {
   describe('on GET', () => {
     it('should remove session cookie', async () => {
       idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+      idamServiceMock.resolveInvalidateSession('ABC')
 
       await request(app)
         .get(AppPaths.logoutReceiver.uri)
         .set('Cookie', `${cookieName}=ABC`)
         .expect(res => expect(res).to.have.cookie(cookieName, ''))
     })
+
+    it('should remove session cookie even when session invalidation is failed ', async () => {
+      idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+      idamServiceMock.rejectInvalidateSession(idamServiceMock.defaultServiceAuthToken)
+
+      await request(app)
+        .get(AppPaths.logoutReceiver.uri)
+        .set('Cookie', `${cookieName}=${idamServiceMock.defaultServiceAuthToken}`)
```

Comment:
It might be misleading to read that service auth token (idamServiceMock.defaultServiceAuthToken) is used in logout. I would be happy for you to rename `idamServiceMock.defaultServiceAuthToken` to `idamServiceMock.defaultAuthToken`

Reply:
done